firmware/Core/Inc/telecommands/freertos_telecommand_defs.h

```diff
@@ -0,0 +1,15 @@
+
+#ifndef __INCLUDE_GUARD__FREERTOS_TELECOMMAND_DEFINITIONS_H
+#define __INCLUDE_GUARD__FREERTOS_TELECOMMAND_DEFINITIONS_H
+
+#include <stdint.h>
+#include <FreeRTOS.h>
+#include <task.h>
```

Comment:
Please include only the bare minimum necessary header files in our `.h` files, and include the rest in the relevant `.c` files. Here, you should only need `stdint.h`, I think.

Reply:
I added the `#include <FreeRTOS.h>` and  `#include <task.h>` because in the `const char* freertos_current_state_enum_to_str(eTaskState state)`  function, I pass in an argument of type `eTaskState` which becomes unknown when I remove those header files. 

Is there another way around this? Unless I create my own custom type and make sure it kinda matches what FreeRTOS has. Idk, feels like a long work around. Let me know what you think

Reply:
Because that function is only used in the one `.c` file, you don't even need to include it in a `.h` file at all

Reply:
Ahh I see, so i wouldn't even need to add it to the header file since it isn't being reused else where.